languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/style.xml

```diff
@@ -14374,7 +14359,7 @@ USA
             <example correction="porter ses fruits|apporter ses fruits|être fructueux"><marker>porter fruit</marker></example>
             <example><marker>porter ses fruits</marker></example>
         </rule>
-        <rule id="CUIR_PATENT" name="cuir patent(e)" tone_tags="clarity">
+        <rule id="CUIR_PATENT" name="cuir patent(e)" tone_tags="general">
```

Comment:
that's a weird rule, basically no opens ever

Reply:
I just want to make the mistake in the editor so it can have at least 1 open :(